package.json

```diff
@@ -46,7 +47,7 @@
   },
   "peerDependencies": {
     "codemirror": "^5.26.0",
-    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^0.14.0"
```

Comment:
@AGS- Thanks for the update. But we released `14.0.0` and you specified `0.14.0`.
More details here: https://medium.com/@leeb/graphql-js-preparing-for-v14-0-0-839f823c144e

Can you please correct it?